File: platforms/python/examples/core/renderer/create_2d_texture.py
r = Renderer()
size = Size.from((256, 256))
pixels = [255u8; (256*256*4) as usize]
tex = futures.executor.block_on(
    r.create_2d_texture(pixels, Some(size), Some(wgpu.TextureFormat.Rgba8UnormSrgb))
).unwrap()

File: platforms/web/examples/core/renderer/create_2d_texture.js
const r = new Renderer();
const size = Size.from((256, 256));
const pixels = [255u8; (256*256*4) as usize];
const tex = futures.executor.blockOn(;
    r.create2dTexture(pixels, Some(size), Some(wgpu.TextureFormat.Rgba8UnormSrgb));
).unwrap();